functionary/schema.py

```diff
@@ -33,26 +288,9 @@ def generate_schema_from_functions(functions: List[Function], namespace="functio
         if parameters is not None:
             schema += " = (_: {\n"
             required_params = parameters.get("required", [])
-            for param_name, param in parameters.get("properties", {}).items():
-                # Param Description
-                description = param.get("description")
-                if description is not None:
-                    schema += f"// {description}\n"
-
-                # Param Name
-                schema += f"{param_name}"
-                if param_name not in required_params:
-                    schema += "?"
-
-                # Param Type
-                param_type = param.get("type", "any")
-                if param_type == "integer":
-                    param_type = "number"
-                if "enum" in param:
-                    param_type = " | ".join([f'"{v}"' for v in param["enum"]])
-                schema += f": {param_type},\n"
-
-            schema += "}) => any;\n\n"
+            info_lines = get_parameter_typescript(parameters.get("properties"), required_params, 0)
```

Comment:
can we rename this variable much more appropriately

Reply:
Each line is a line of code in typescript, 
so I change to: tp_lines ?